docs/articles/layouts.md

```diff
@@ -226,52 +231,51 @@ file, as it was mentioned earlier.
 What is important here, is that all the templates can still be natural
 templates and can be viewed in a browser without a running server.
 
-### Including with DOM Selectors
+### Including with Markup Selectors
 
 In Thymeleaf, fragments don't need to be explicitly specified using
 `th:fragment` at the page they are extracted from. Thymeleaf can select
 an arbitrary section of a page as a fragment (even a page living on an
-external server) by means of its DOM Selector syntax, similar to XPath
-expressions and CSS selectors.
+external server) by means of its Markup Selector syntax, similar to XPath
+expressions, CSS or jQuery selectors.
 
 ```xml
-<div th:include="http://www.thymeleaf.org :: p.notice" >...</div>
+<p th:insert="http://www.thymeleaf.org :: section.description" >...</p>
 ```
 
-The above code will include a paragraph with `class="notice"` from
-`thymeleaf.org`. In order to make it happen, the template engine must be
-configured with `UrlTemplateResolver`:
+The above code will include a `section` with `class="description"` from
```

Comment:
Actually, although Thymeleaf would execute this perfectly fine, I believe this example is not correct from an HTML perspective, as it would result in a `<section>` tag being included inside a `<p>`, which is not valid HTML. Perhaps simply switching the tag with the `th:insert` back to a `<div>` would do?